includes/class-wc-amazon-payments-advanced-merchant-onboarding-handler.php

```diff
@@ -32,8 +32,6 @@ class WC_Amazon_Payments_Advanced_Merchant_Onboarding_Handler {
 	public function __construct() {
```

Comment:
Shouldn't this file goes to admin folder too? since all the process is on admin side.

Reply:
Would make sense to have it there yeah. I'd need to evaluate the implications. But thanks for the suggestion!